### ISSUE TITLE
[Arc] Failed to Fetch Data Error

### ISSUE THREAD

**Issue by abass** [raycast/extensions#5165]
### Extension

https://www.raycast.com/the-browser-company/arc

### Description

The "Search Tabs" feature just totally stopped working:
<img width="750" alt="CleanShot 2023-03-05 at 17 42 43@2x" src="https://user-images.githubusercontent.com/5505994/222992789-0f28efbc-73e3-4d52-9ba6-2ffbf3b6d9e1.png">

### Steps To Reproduce

Search tabs, you'll get an error:
<img width="750" alt="CleanShot 2023-03-05 at 17 42 43@2x" src="https://user-images.githubusercontent.com/5505994/222992800-7fa52efc-89ff-40a8-b8bd-03fdc876e9fe.png">

### Current Behaviour

It just errors out

### Expected Behaviour

It used to actually give the tab results

**Comment by thomaspaulmann** [comment]
Looks like you have a tab with an unusual character open. Could you check if any of the tabs has a special title and/or url? That would help with debugging.

**Comment by abass** [comment]
Hmm, super strange! I don't see any tabs open with odd characters. Even
closing out of Arc completely and reopening it, the same issue happens.

If you're talking bookmarks (pinned/favorites tabs) then oh gosh...

The only characters I use a lot are:
—
–
-
And maybe some:
→

Any chance any of those could be problematic?

**Comment by thomaspaulmann** [comment]
Yeah that could be it... they might need some additional decoding. I wonder what's the best way to debug this is. Maybe we can add some way to log the tab titles in the error message?

**Comment by abass** [comment]
It should stay open, it's still broken 😢

--
Alex Bass
*Founder & Head of Product*
Efficient App ( https://efficient.app/?ref=email-signature ) | Efficient VC ( https://efficientvc.com/?ref=email-signature )

Connect with me:
read.cv ( https://alexbass.xyz/?ref=email-signature ) | LinkedIn ( https://linkedin.com/in/alexhbass ) | Twitter ( https://twitter.com/alexhbass ) | YouTube ( https://efficient.link/youtube ) | Subscribe to my Journey ( https://digest.efficientvc.com/?ref=email-signature )

Looking for Inspo? Check out other CRM Integrations ( https://efficient.app/app-category/crm?ref=email-signature ) that Efficient App customers are using.

Know a company that can use our help? Send them a Free Efficiency Audit ( https://efficient.app/audit?ref=email-signature ) 🚀

Typed joyfully with Canary ( https://efficient.link/canary )

Sent via Superhuman ( https://efficient.link/r/superhuman )

**Comment by abass** [comment]
Bummer, it's still not fixed :/

--
Alex Bass
*Founder & Head of Product*
Efficient App ( https://efficient.app/?ref=email-signature ) | Efficient VC ( https://efficientvc.com/?ref=email-signature )

Connect with me:
read.cv ( https://alexbass.xyz/?ref=email-signature ) | LinkedIn ( https://linkedin.com/in/alexhbass ) | Twitter ( https://twitter.com/alexhbass ) | YouTube ( https://efficient.link/youtube ) | Subscribe to my Journey ( https://digest.efficientvc.com/?ref=email-signature )

Looking for Inspo? Check out other CRM Integrations ( https://efficient.app/app-category/crm?ref=email-signature ) that Efficient App customers are using.

Know a company that can use our help? Send them a Free Efficiency Audit ( https://efficient.app/audit?ref=email-signature ) 🚀

Typed joyfully with Canary ( https://efficient.link/canary )

Sent via Superhuman ( https://efficient.link/r/superhuman ) (get 2 free months)

**Comment by abass** [comment]
Still not fixed 🫣

--
Alex Bass
*Founder & Head of Product*
Efficient App ( https://efficient.app/?ref=email-signature ) | Efficient VC ( https://efficientvc.com/?ref=email-signature )

Connect with me:
read.cv ( https://alexbass.xyz/?ref=email-signature ) | LinkedIn ( https://linkedin.com/in/alexhbass ) | Twitter ( https://twitter.com/alexhbass ) | YouTube ( https://efficient.link/youtube ) | VC Journey ( https://efficientvc.com/?ref=email-signature )

Looking for software inspiration? Check out our new recommended software stacks ( https://efficient.app/stacks?ref=email-signature ) and top picks ( https://efficient.app/top-picks?ref=email-signature ) !

Know a company that can use our help? Send them a Free Efficiency Audit ( https://efficient.app/audit?ref=email-signature ) 🚀

Typed joyfully with Canary ( https://efficient.link/canary )

Sent via Superhuman ( https://efficient.link/r/superhuman ) (get 1 free month)

**Comment by QIanGua** [comment]
Get same issue many times!

**Comment by QIanGua** [comment]
Hi @thomaspaulmann , I find an testing URL that it failed to parse its title. [Anthropic \\ Home](https://www.anthropic.com/)

title: Anthropic \\ Home
url: https://www.anthropic.com/

**Comment by canada4663** [comment]
+1, seeing similar issue with sites with \ in the title
